### PR TITLE
Small polishes

### PR DESCRIPTION
**Pull request by FeroTheFox**
- Adjusts button paddings to be larger
- Renames "Earliest" to "Oldest" in dropdowns
- Adds gaps between cards on the discover page in mobile breakpoint

### CODE REVIEW

File: lib/banchan_web/live/discover_live/components/offerings.ex [hunk 1]
```diff
@@ -78,7 +78,7 @@ defmodule BanchanWeb.DiscoverLive.Components.Offerings do
           {/if}
         </div>
       {#else}
-        <div class="grid grid-cols-1 offering-list sm:gap-2 sm:grid-cols-2 lg:grid-cols-3 auto-rows-fr">
+        <div class="grid grid-cols-1 offering-list gap-2 sm:gap-2 sm:grid-cols-2 lg:grid-cols-3 auto-rows-fr">
```
Comment (zkat) [inline lib/banchan_web/live/discover_live/components/offerings.ex:81]: Maybe you’ll want to get rid of the `sm:gap-2`?

Comment (FeroTheFox) [reply lib/banchan_web/live/discover_live/components/offerings.ex:81]: Ah yea, good catch

**Comment by zkat** [comment]
I'll let @rjt-rockx  give the final +1, since y'all are coordinating on this, but this looks fine to me.

**Comment by rjt-rockx** [comment]
lgtm

**Pull request merged**